### PR TITLE
qa/tests: added quincy and removed master runs

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Review by neha-ojha — COMMENTED** [review]
@yuriw we should add `-s orch` too because of https://github.com/ceph/ceph/pull/44828
FYI @adk3798

**Comment by yuriw** [comment]
@neha-ojha added `orch`

**Review by neha-ojha — APPROVED** [review]

**Review by adk3798 — APPROVED** [review]

File: qa/crontab/teuthology-cronjobs [hunk 1]
```diff
@@ -36,20 +36,21 @@ CEPH_QA_EMAIL="ceph-qa@ceph.io"
 0 8  * * 5 CEPH_BRANCH=octopus; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -m $MACHINE_NAME -s smoke -k distro -e $CEPH_QA_EMAIL -p 70
 7 8  * * 6 CEPH_BRANCH=pacific; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -m $MACHINE_NAME -s smoke -k distro -e $CEPH_QA_EMAIL -p 70
 
-## master branch runs - weekly
+## quincy branch runs - weekly
```
Comment (batrick) [inline qa/crontab/teuthology-cronjobs:39]: I think master runs are still useful for a baseline. Can we make them at least monthly, maybe with a higher priority too?

Also, for new releases we usually run ~2 times a week, which I think is appropriate. Let's do that too?

**Review by batrick — CHANGES_REQUESTED** [review]

**Comment by yuriw** [comment]
jenkins test make check

**Review by batrick — CHANGES_REQUESTED** [review]
I think we need to reevaluate priority assignment for release branches:

* `pacific` to `-p 750`
* `quincy` to `-p 150` (for new release, eventually `-p 500`?)
* weekly `master` at default priority (`-p 1000`) ???

Remove `octopus` / `nautilus`.

I still think it's valuable to do regular baseline runs for master. Maybe monthly with high priority, like `-p 100`.

File: qa/crontab/teuthology-cronjobs (hunk 1 above)
Comment (cbodley) [reply qa/crontab/teuthology-cronjobs:39]: the last manual master baseline i saw was from feb 2nd in 
http://pulpito.ceph.com/?branch=wip-yuriw-master-2.1.22

i agree with @batrick, i'd prefer to find master runs in pulpito if we have the resources for it

Comment (batrick) [reply qa/crontab/teuthology-cronjobs:39]: Is there a reason the `master` branch name can't be used in these baseline runs? A `-S` can be used for a specific revision. Also a `--suite-branch` with the same revision so they are in sync. (You can't specify the revision for the `--suite-branch` like you can for `--ceph`.)

Comment (neha-ojha) [reply qa/crontab/teuthology-cronjobs:39]: It can absolutely be used, @yuriw can you please keep this in mind.

Comment (jdurgin) [reply qa/crontab/teuthology-cronjobs:39]: sounds like using the master branch name for the weekly baseline runs would give us the coverage and discoverability we're looking for

**Comment by neha-ojha** [comment]
> I think we need to reevaluate priority assignment for release branches:
> 
> * `pacific` to `-p 750`
> * `quincy` to `-p 150` (for new release, eventually `-p 500`?)
> * weekly `master` at default priority (`-p 1000`) ???
> 
> Remove `octopus` / `nautilus`.
> 
> I still think it's valuable to do regular baseline runs for master. Maybe monthly with high priority, like `-p 100`.

@batrick we can make these changes in a follow-up PR, mind unblocking this one?

**Pull request merged**